tests/integration/pkg/tests/custom/custom_test.go

```diff
@@ -32,6 +33,9 @@ func TestSystemAgentVersion(t *testing.T) {
 }
 
 func TestCustomOneNode(t *testing.T) {
+	if strings.ToLower(os.Getenv("DIST")) == "rke2" {
+		return
```

Comment:
I think it's better to use `t.Skip()` for these to make it explicit. If we are actually going to do this.